source/frontend/src/components/measurements/AreaForm.tsx

```diff
@@ -1,17 +1,17 @@
 import { AreaUnitTypes } from 'constants/index';
-import { TableCaption } from 'features/mapSideBar/tabs/SectionStyles';
 import React, { useCallback, useEffect, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
 import { convertArea, round } from 'utils';
 
-import { StyledTable } from '../styles';
+import { StyledGreenCol, StyledGreenGrey, StyledInput } from './styles';
 
-export interface IUpdateLandMeasurementEditTableProps {
+export interface IAreaFormProps {
   area?: number;
   areaUnitTypeCode?: string;
-  onChange?: (landArea: number, areaUnitTypeCode: string) => void;
+  onChange: (landArea: number, areaUnitTypeCode: string) => void;
```

Comment:
Not a big deal but this change is forcing the consumer to **always** pass an _onChange_ event handler. There might be cases where the consumer does not care about the changes, it only wants to render the table

Reply:
I think this is better given the parent using the container can just pass an empty function. Otherwise the child needs to know how it could be used instead of assuming the parent will figure out